tests/DecisionServiceTests/DecisionServiceTest.php

```diff
@@ -118,6 +118,55 @@ public function testGetVariationBucketsUserWhenExperimentIsRunning()
         );
     }
 
+    public function testGetBucketingIdBucketingIdAttributeNotString()
+    {
+        $decisionService = new DecisionTester($this->loggerMock, $this->config, $this->userProvideServiceMock);
+        $userAttributesWithBucketingId = [
+            'device_type' => 'iPhone',
+            'company' => 'Optimizely',
+            'location' => 'San Francisco',
+            '$opt_bucketing_id' => 5
+        ];
+
+        $this->loggerMock->expects($this->never())
+            ->method('log');
+
+        $this->assertSame($decisionService->getBucketingId($this->testUserId, $userAttributesWithBucketingId), $this->testUserId);
```

Comment:
Our convention is to put expected value in assertion as the first argument. Do this for others as well